doc/ceph-volume/lvm/prepare.rst

```diff
@@ -29,50 +29,58 @@ Although :term:`bluestore` is the default, the back end can be specified with:
 
 ``bluestore``
 -------------
-The :term:`bluestore` objectstore is the default for new OSDs. It offers a bit
-more flexibility for devices compared to :term:`filestore`.
-Bluestore supports the following configurations:
-
-* A block device, a block.wal, and a block.db device
-* A block device and a block.wal device
-* A block device and a block.db device
-* A single block device
-
-The bluestore subcommand accepts physical block devices, partitions on
-physical block devices or logical volumes as arguments for the various device parameters
-If a physical device is provided, a logical volume will be created. A volume group will
-either be created or reused it its name begins with ``ceph``.
-This allows a simpler approach at using LVM but at the cost of flexibility:
-there are no options or configurations to change how the LV is created.
+:term:`Bluestore<bluestore>` is the default object store for new OSDs. It
```

Comment:
Suggest "back end" instead of "object store".  The more we talk about RADOS objects, the more people get confused wrt S3 objects.

Reply:
See https://github.com/ceph/ceph/pull/48644.